bluesky_queueserver/tests/test_examples.py

```diff
@@ -1,3 +0,0 @@
-def test_one_plus_one_is_two():
-    "Check that one and one are indeed two."
-    assert 1 + 1 == 2
```

Comment:
Should this file be removed completely?

Reply:
Yes. I will delete it as part of the next PR. I thought about leaving the file for future tests without looking at the name.

Reply:
It is the cookie-cutter dummy test, seems ok to nuke to me.